src/commands/user/level.ts

```diff
@@ -38,60 +30,58 @@ export default class Level {
     const selectedMember = guild.members.cache.filter((member) => member.user.id === id).map((user) => user)[0];
     const avatarURL = selectedMember.displayAvatarURL();
 
-    const levelEnabled = await settingsTable?.get(`${guild.id}.leveling.enabled`).then(
-      (data) => {
+    const levelEnabled = await settingsTable
+      ?.get(`${guild.id}.leveling.enabled`)
+      .then(data => {
         if (!data) return false;
-        return Boolean(data);
-      }
-    ).catch(() => false);
-    if (!levelEnabled) {
-      return await interaction.followUp({
-        embeds: [errorEmbed("Leveling is disabled for this server.")]
-      });
-    }
+        return data;
+      })
+      .catch(() => false);
 
-    const { exp, levels } = await levelingTable?.get(`${guild.id}.${selectedMember.id}`).then(
-      (data) => {
+    if (!levelEnabled) return await interaction.followUp({
+      embeds: [errorEmbed("Leveling is disabled for this server.")]
+    });
+
+    const { exp, levels } = await levelingTable
+      ?.get(`${guild.id}.${selectedMember.id}`)
```

Comment:
```suggestion
    const { exp, levels } = await levelingTable?
      .get(`${guild.id}.${selectedMember.id}`)
```
That character belongs there I think ;)